### ISSUE TITLE
Some image alt texts in the tutorial need to be updated, since they are just "alt text"

### ISSUE THREAD

**Issue by evertrol** [NLeSC/spot#183]
Some of the alternate text of images in the tutorial (perhaps elsewhere as well) are simply "alt text". Good practice dictates these should be more meaningful, preferably indicating what they represent (not so much what the picture is).

Naturally, for SPOT, this isn't the most important thing, since SPOT is for a large part a visual tool; someone using a screen reader will find little utility in SPOT.

**Comment by evertrol** [comment]
Moved to https://github.com/NLeSC/spot-tutorial/issues/7
